src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.cs

```diff
@@ -1453,7 +1459,7 @@ internal SqlException() { }
         /// <include file='../../../../doc/snippets/Microsoft.Data.SqlClient/SqlException.xml' path='docs/members[@name="SqlException"]/State/*'/>
         public byte State { get { throw null; } }
 
-        /// <include file='../../../../doc/snippets/Microsoft.Data.SqlClient/SqlException.xml' path='docs/members[@name="SqlException"]/GetObjectData/*'/>
+        /// <inheritdoc cref="System.Exception.GetObjectData" />
```

Comment:
I've heard inhericdoc doesn't work across doc repositories. Does this appear correctly in the IDE?

Reply:
Hmm. It shows up correctly in the IDE, but if @JRahnama is correct about how it gets generated into doc websites, then we should probably not use this.

Reply:
Also for the record, the xml link was to an inheritdoc tag, which for whatever reason doesn't show up correctly in the IDE. I can replace it with a "to be added" summary if that's ok?

Reply:
It looks like inheritdoc support may have actually been added to the API doc generation tool in the last few years. I vote leave it and we'll see how it turns out.

Reply:
We can add them and request a build from the documentation team on a test branch to review the results and determine whether they should remain.

I remember that for GetObjectData, everything worked fine with no issues. However, in the SqlBatch area and related classes, we encountered many cases where we ended up duplicating code.